inc/functions_modcp.php

```diff
@@ -181,11 +181,36 @@ function send_report($report)
 		}
 	}
 
-	while($mod = $db->fetch_array($query))
+	$lang_string_subject = "emailsubject_report{$report_type}";
+	$lang_string_message = "email_report{$report_type}";
+
+	if(!$lang_string_subject || !$lang_string_message)
```

Comment:
What about them? Even if there are 100 plugins installed that change report types `$lang_string_subject` and `$lang_string_message` will be always filled in the 2 lines above.. So that check doesn't do anything useful.

It's something like:
```php
$yes = true;

if($yes === false)
{
    //...
}
```